admin-dev/themes/default/sass/partials/_multistore.sass

```diff
@@ -34,37 +34,56 @@
 
 #header_shop
   .dropdown-menu
-    min-width: 210px
-    list-style: none
+    margin: 0
+    padding: 0
+    border-radius: 0
+    min-width: 17.75rem !important
+    box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.1)
+    border: 1px solid #bbcdd2
```

Comment:
Border color can be a variable because used many times.

Reply:
Yeah, I don't think it's really important right now though.